js/samples/menu-example/basic/basic/src/index.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This sample is referenced by the genkit docs. Changes should be made to
+// both.
+import { generate } from '@genkit-ai/ai';
+import { configureGenkit } from '@genkit-ai/core';
+import { defineFlow, startFlowsServer } from '@genkit-ai/flow';
+import { geminiPro, googleAI } from '@genkit-ai/googleai';
+import * as z from 'zod';
+
+configureGenkit({
+  plugins: [googleAI()],
+  logLevel: 'debug',
+  enableTracingAndMetrics: true,
+});
+
+export const menuQAFlow = defineFlow(
+  {
+    name: 'menuQAFlow',
+    inputSchema: z.string(),
+    outputSchema: z.string(),
+  },
+  async (subject) => {
+    const llmResponse = await generate({
+      model: geminiPro,
+      prompt: `Our menu today includes burgers, spinach, and cod.
+      Tell me if ${subject} can be found on the menu`,
```

Comment:
I am not 💯 on how it ties in with your broader plans, but what we discussed in-person was perhaps turning this into something more creative/generative in nature, like a menu generator, or similar.